doc/whats_new/v0.23.rst

```diff
@@ -89,8 +89,12 @@ Changelog
 ...............................
 
 - |Enhancement| :func:`gaussian_process.kernels.Matern` returns the RBF kernel when ``nu=np.inf``.
-  :pr:`15503` by :user:`Sam Dixon <sam-dixon>`.
-  
+  :pr: `15503` by :user:`Sam Dixon` <sam-dixon>.
+
+- |Fix| Fixed bug in :class:`gaussian_process.GaussianProcessRegressor` that
+  caused predicted standard deviations to only be between 0 and 1. :pr:`15782`
```

Comment:
if this was only an issue when normalize=True, please say so,.

Reply:
So you'd still have predicted variance between [0, 1] whether or not the data is normalised - normalising the data is the solution. The only exception to this is if WhiteKernel is being used, but that would only by applied if the training data was noisy (measurement noise etc.) 

I made a small change to the text.